docs/guides/quick-start.md

```diff
@@ -46,7 +46,7 @@ from .types import Fruit
 
 @strawberry.type
 class Query:
-    fruits: List[Fruit]
+    fruits: List[Fruit] = strawberry_django.field()
```

Comment:
this shouldn't be required, was it throwing an error?

Reply:
@patrick91 I am really sorry about this mistake :(
I sent this request long ago before I became more expert on strawberry options
So i close this PR